massa-versioning-worker/src/versioning_ser_der.rs

```diff
@@ -589,13 +591,196 @@ impl Deserializer<MipState> for MipStateDeserializer {
 
 // End MipState
 
+// MipStoreStats
+
+/// Serializer for `VersioningStoreRaw`
+pub struct MipStoreStatsSerializer {
+    u32_serializer: U32VarIntSerializer,
+    u64_serializer: U64VarIntSerializer,
+}
+
+impl MipStoreStatsSerializer {
+    /// Creates a new `Serializer`
+    pub fn new() -> Self {
+        Self {
+            u32_serializer: U32VarIntSerializer::new(),
+            u64_serializer: U64VarIntSerializer::new(),
+        }
+    }
+}
+
+impl Default for MipStoreStatsSerializer {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl Serializer<MipStoreStats> for MipStoreStatsSerializer {
+    fn serialize(&self, value: &MipStoreStats, buffer: &mut Vec<u8>) -> Result<(), SerializeError> {
+        // config
+        let cfg_1 = u32::try_from(value.config.block_count_considered).map_err(|e| {
+            SerializeError::GeneralError(format!("Could not convert to u32: {}", e))
+        })?;
+        let cfg_2 = u32::try_from(value.config.counters_max).map_err(|e| {
+            SerializeError::GeneralError(format!("Could not convert to u32: {}", e))
+        })?;
```

Comment:
Do you think it makes sense to serialize the config values along the struct? Usually in the codebase it's serialized as is and the config is passed in the deserializer args to check the validity of the structure. But maybe here it makes sense.

Reply:
Removed cfg ser / der :)